stories/Truncate.stories.js

```diff
@@ -10,52 +10,67 @@ const limit = 10
 
 stories.add('default', () => (
   <div>
-    <p>
-      Auto:<br />
+    <p style={{ width: '50%' }}>
+      Auto:
+      <br />
```

Comment:
This file auto-formatted to move the `<br />` to the next line -- LMK if I should change this back/modify my formatter.